### DISCUSSION TITLE
Adding items to libraries manually

### DISCUSSION THREAD

**Discussion by overtoneblue** [itsToggle/plex_debrid#118] [Q&A] [answered]
Is there a proper way to add items manually into the libraries "movies", and "shows" when using the mounted drive through rclone? whenever I try to add something manually it either doesnt work, or when adding a standalone file, when I need to reboot the host machine rclone has errors because the cached files are not found. I'm on arch linux running my plex server, jackett, rclone, etc. all from this machine

**Comment by itsToggle** [comment, marked as answer]
The mounted drive is read-only, so you can't "upload" anything to realdebrid.
What you can do is manually add a torrent file or magnet link on the realdebrid website. You can also use plex_debrid to manually add content by scraping for a movie/show from the main menu scraper.

**Reply by itsToggle** [reply]
You could add quite a few public indexers to jackett (like rarbg, 1337x, thepiratebay, eztv, yts, etc.). Once you added jackett, you can remove the "legacy" scrapers for 1337x and rarbg from the plex_debrid scraper sources, as scraping times will decrease when only jackett is used.
Aside from that there is not much else you can improve - If there is no multi-season pack for a show, plex_debrid will try to download single season packs. If no single season pack for a season is available, plex_debrid will try to download episodes. The wait time will increase accordingly. 
The only way to improve wait times for tv shows is making sure plex_debrid is able to find season packs - which you can do by increasing the quality and quantity of the torrent indexers you add to Jackett :)

**Comment by jamespdaily** [comment]
Sorry to necro this, but I'm stuck at this step too and not sure I understand.

I have everything configured with docker and get the same error:

[14/01/23 17:43:26] [plex error]: Your library seems empty. To prevent unwanted behaviour, no further downloads will be started. If your library really is empty, please add at least one media item manually.

So i found a random TV episode on RARBG and uploaded to my Realdebrid account, however I'm not seeing anything happening. I think im confused how to get this random episode to my library

**Reply by itsToggle** [reply]
did you refresh your plex library?
